tools/clang/unittests/SPIRV/TestMain.cpp

```diff
@@ -103,9 +103,6 @@ int main(int argc, char **argv) {
       // Allow the user set the root directory for test input files.
       if (i + 1 < argc) {
         clang::spirv::testOptions::inputDataDir = argv[++i];
-      } else {
```

Comment:
This check should stay, since it's only triggered if somebody passes `--spirv-test-root` without a following argument. It should be an error not providing an argument to the option (if somebody goes as far as providing the option)

Reply:
doh... of course.